packages/parser/src/CombatUnit.ts

```diff
@@ -27,7 +27,7 @@ export interface ICombatUnit {
   info?: CombatantInfo;
 
   damageIn: CombatHpUpdateAction[];
-  damageOut: CombatHpUpdateAction[];
+  damageOut: (CombatHpUpdateAction | CombatAbsorbAction)[];
```

Comment:
interesting.. does all downstream logic handle this well? I'm pleasantly surprised..

Reply:
are we handling partial absorbs well? (dealt 10k, 5k absorbed, 5k effective)